optimum/neuron/utils/hub_cache_utils.py

```diff
@@ -78,8 +78,9 @@ def create_compile_cache():
     "sample_size",
     "projection_dim",
     "_use_default_values",
+    "_attn_implementation_autoset",
 ]
-NEURON_CONFIG_WHITE_LIST = ["input_names", "output_names", "model_type"]
+NEURON_CONFIG_WHITE_LIST = ["input_names", "output_names", "model_type", "static__commit_hash"]
```

Comment:
Not sure, but shouldn't this be
```suggestion
NEURON_CONFIG_WHITE_LIST = ["input_names", "output_names", "model_type", "static_commit_hash"]
```

Reply:
Good catch!!! it comes from latest transformers, I should have add the arg instead of just adding it to the white list. Thanks Alvaro!